lib/generators.nix

```
@@ -243,7 +243,7 @@ rec {
   toYAML = {}@args: toJSON args;
```

Isn't this noop, too?

---

could be changed to but I don't know if it's necessary
```suggestion
  toYAML = {}: toJSON {};
```